### ISSUE TITLE
Make import\export for tox profile

### ISSUE THREAD

**Issue by dev-dvl** [Zoxcore/trifa_material#253]
Subject

**Comment by akontsevich** [comment]
Hi @zoff99, why You marked this ticket as invalid? I tried TRIfA Android version and it created new TOX ID on start, why we can't import our existing TOX profile on start which was available in very early versions of uTox/qTox, etc?

**Comment by akontsevich** [comment]
Related issues:
- https://github.com/zoff99/ToxAndroidRefImpl/issues/73#issuecomment-2498882468
- https://github.com/Zoxcore/trifa_material/issues/253
- https://github.com/zoff99/ToxAndroidRefImpl/issues/19
- etc

**Comment by akontsevich** [comment]
> trifa desktop does not need any import or export function. the toxsave data file is just a file. you can copy it.

Good! :slightly_smiling_face: I'll try it. What about Android version? @zoff99 may I create ticket for this? aTox allows to import `name.tox` on the first start and only 1 export item in menu - works just fine. Same for uTox/qTox, etc. Why do you have 3 import/export options in TRIfA? It is very confusing! Can we have standard feature for Tox clients, please? Import/Export only 1 `*.tox` file? Thanks!

**Comment by akontsevich** [comment]
> trifa desktop does not need any import or export function. the toxsave data file is just a file. you can copy it.

Wrong! I tried it. Both uTox and qTox use `~/.config/tox` path so we can select necessary profile on start. TRIfA starts with `~/.local/share/trifa_material` so need to manually copy/rename file which is also unclear as uTox and qTox both use `name.tox` file, while you use `toxsave.tox`. So it is not clear what file to replace/rename, etc.

Please implement normal standard feature.
